Mathlib/LinearAlgebra/Matrix/Adjugate.lean

```diff
@@ -417,6 +417,210 @@ theorem adjugate_fin_two (A : Matrix (Fin 2) (Fin 2) α) :
 theorem adjugate_fin_two_of (a b c d : α) : adjugate !![a, b; c, d] = !![d, -b; -c, a] :=
   adjugate_fin_two _
 #align matrix.adjugate_fin_two_of Matrix.adjugate_fin_two_of
+--adjugate !![a, b, c; d, e, f; g, h, i]
+theorem adjugate_fin_three (A : Matrix (Fin 3) (Fin 3) α) :
+    adjugate A =
+    !![(A 1 1) * (A 2 2) - (A 1 2) * (A 2 1),
+     -((A 0 1) * (A 2 2)) + (A 0 2) * (A 2 1),
+      (A 0 1) * (A 1 2) - (A 0 2) * (A 1 1);
+     -((A 1 0) * (A 2 2)) + (A 1 2) * (A 2 0),
+      (A 0 0) * (A 2 2) - (A 0 2) * (A 2 0),
+       -((A 0 0) * (A 1 2)) + (A 0 2) * (A 1 0);
+      (A 1 0) * (A 2 1) - (A 1 1) * (A 2 0),
+       -((A 0 0) * (A 2 1)) + (A 0 1) * (A 2 0),
+        (A 0 0) * (A 1 1) - (A 0 1) * (A 1 0)] := by
+
+    ext h1 h2
+    fin_cases h1
```

Comment:
Whenever you  use a tactic like `fin_cases` that creates multiple goals, you should mark each goal with a bullet point, as
```lean
fin_cases i
· sorry
· sorry
· sorry
```
where the three `sorry`s will be pieces of your proof below.

Reply:
I will work on this later (when school is over lol). How would you shorten the proof?